e2e/README.md

```diff
@@ -7,6 +7,27 @@ These tests are run against public testnets: Ropsten and Rinkeby.
 
 e2e package contains a few utilities which are described in a [godoc](https://godoc.org/github.com/status-im/status-go/e2e).
 
+### Flags
+
+#### 1. `-network`
+The `-network` flag is used to provide either a network id or network name which specifies the ethereum network to use
+for running all test. It by default uses the `StatusChain` network.
+
+#### Usage
+
+To use the `ropsten` network for testing using network name:
+
+```bash
+go test -v ./e2e/... -network=rinkeby
```

Comment:
The example is iconsistent. It says `ropsten` above while the network is `rinkeby`.